src/server/data/automatedTestingData.js

```diff
@@ -13,6 +13,7 @@ const { getConnection } = require('../db');
 const { redoCik } = require('../services/graph/redoCik');
 const { refreshAllReadingViews } = require('../services/refreshAllReadingViews');
 const fs = require('fs').promises;
+const conn = getConnection();
```

Comment:
In every case I checked, OED uses ``getConnection()`` within a function and not outside. I think this is better practice so this should be added where needed within functions.